examples/recipes/create_subplots.py

```diff
@@ -24,7 +24,7 @@
 # Fernando Perez has provided a nice top level method to create in
 # :func:`~matplotlib.pyplots.subplots` (note the "s" at the end)
 # everything at once, and turn on x and y sharing for the whole bunch.
-# You can either unpack the axes individually::
+# You can either unpack the axes individually.
```

Comment:
While you're cleaning this up...any chance to make that an actual sentence--or make some semblance of sense?

Reply:
Oh, I see now...probably should end in "..." to indicate the continuation coming later.